pkg/chunk/schema_config.go

```diff
@@ -109,6 +111,7 @@ func (cfg *SchemaConfig) loadFromFile() error {
 // Validate the schema config and returns an error if the validation
 // doesn't pass
 func (cfg *SchemaConfig) Validate() error {
+
```

Comment:
This is nit. Generally speaking, we try to preserve the rest of the codebase as is, without adding non-functional changes to blocks of that haven't changed.

Reply:
I will keep in mind about preserving the rest of the codebase.